STX.Serialization.Providers.Abstractions.Tests.Unit/SerializationAbstractionProviderTests.Logic.Serialize.cs

```diff
@@ -0,0 +1,41 @@
+// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace LHDS.Core.Tests.Unit.Services.Foundations.Addresses
+{
+    public partial class SerializationAbstractionProviderTests
+    {
+        [Fact]
+        public async Task ShouldSerializeAsync()
+        {
+            // given
+            dynamic dynamicPerson = new
```

Comment:
If we don't really care about this object or it's dynamic model then should we call it `somePerson`?